simplicity-sys/README.md

```diff
@@ -0,0 +1,19 @@
+simplicity-sys
+=============
+
+
+This crate provides Rust definitions for the FFI structures and methods.
+
+
+## Vendoring
+
+The default build process is to build using the vendored libsecp256k1 sources in
+the depend folder. There is no support for symbols yet.
+
+To update the vendored sources, use the `vendor-simplicity.sh` script:
+
+```
+$ ./vendor-libsecp.sh depend <rev>
```

Comment:
In 27cbcfb96e0bc48dec0951661b93695f80ef1531:

`vendor-libsecp` should be `vendor_simplicity.sh`. (Also, consider renaming vendor_simplicity to vendor-simplicity.)